program/Project Descriptions and Plans/SDK/sdk-project-concept.md

```diff
@@ -112,16 +119,17 @@ The following diagram shows the relationship of the SDK to the parent project.
 
 The SDK project is one of the components that feeds into the complete product. In summary:
 
-| Task                    | Lead organization  | Gap analysis       |
-|-------------------------|--------------------|------------------- |
-| OpenHW IDE and debugger | Alexander Fedorov? | 4 eng. months      |
-| GNU compiler tool chain | Embecosm           | -                  |
-| HAL                     | Alibaba T-Head     | N/A for release 1  |
-| FreeRTOS                | ?                  | 7 eng. months      |
-| Symbiflow tooling       | ?                  | -                  |
-| Documentation           | ?                  | 6 - 12 eng. months |
-| Examples/applications   | ?                  | 12 eng. months     |
-| Integration & installer | Ashling (lead)     | 6 eng. months      |
+| Task                       | Lead organization  | Gap analysis           |
+|----------------------------|--------------------|----------------------- |
+| OpenHW IDE and debugger    | Alexander Fedorov? | 4 engineer months      |
```

Comment:
Can we mention ArSysOp ([Contributing Member of Eclipse Foundation](https://www.eclipse.org/membership/showMember.php?member_id=1467)) here instead of "Alexander Fedorov"

Reply:
Done.